experimental/dds/tree2/src/test/feature-libraries/flex-tree/events.spec.ts

```diff
@@ -656,6 +656,55 @@ describe("beforeChange/afterChange events", () => {
 		assert.strictEqual(childBeforeCounter, 1);
 		assert.strictEqual(childAfterCounter, 1);
 	});
+
+	it("stopPropagation works", () => {
```

Comment:
Since this PR also fundamentally moves propagation handling to the FlexTree layer, are there any more tests we should add here? Or is the coverage already there?

Reply:
I think we're good. The whole suite from lines 17 to 708 is dedicated to tests for beforeChange/afterChange on the FlexTree layer.